src/io/flutter/jxbrowser/EmbeddedBrowser.java

```diff
@@ -20,19 +21,30 @@
 import java.util.concurrent.atomic.AtomicBoolean;
 
 import static com.teamdev.jxbrowser.engine.RenderingMode.HARDWARE_ACCELERATED;
+import static com.teamdev.jxbrowser.engine.RenderingMode.OFF_SCREEN;
 
 public class EmbeddedBrowser {
   public void openPanel(ContentManager contentManager, String tabName, String url) {
-    final EngineOptions options =
+    EngineOptions options =
       EngineOptions.newBuilder(HARDWARE_ACCELERATED).build();
-    final Engine engine = Engine.newInstance(options);
-    final Browser browser = engine.newBrowser();
-    browser.settings().enableTransparentBackground();
+    Engine engine = Engine.newInstance(options);
+    Browser browser = engine.newBrowser();
+
+    try {
+      browser.settings().enableTransparentBackground();
+    } catch (UnsupportedRenderingModeException ex) {
+      options =
+        EngineOptions.newBuilder(OFF_SCREEN).build();
+      engine = Engine.newInstance(options);
+      browser = engine.newBrowser();
+      browser.settings().enableTransparentBackground();
+    }
 
     // Multiple LoadFinished events can occur, but we only need to add content the first time.
     final AtomicBoolean contentLoaded = new AtomicBoolean(false);
 
     browser.navigation().loadUrl(url);
+    final Browser finalBrowser = browser;
```

Comment:
nit: having a variable named `finalBrowser` is a bit ugly.  A slightly cleaner option would be to have a `createBrowser()`helper method.
`final Browser browser= createBrowser(..)`

Reply:
If I just skip the transparent browser step, then we don't really need a separate function here.